vagrant/boxes.d/00-centos.yaml

```diff
@@ -7,3 +7,12 @@ boxes:
     scenarios:
       - foreman
       - katello
+
+  centos10-stream:
+    box_name: 'centos/stream10'
+    disk_size: 40
+    libvirt: https://composes.stream.centos.org/stream-10/production/latest-CentOS-Stream/compose/BaseOS/x86_64/images/CentOS-Stream-Vagrant-10-20241120.0.x86_64.vagrant-libvirt.box
+    virtualbox: https://composes.stream.centos.org/stream-10/production/latest-CentOS-Stream/compose/BaseOS/x86_64/images/CentOS-Stream-Vagrant-10-20241120.0.x86_64.vagrant-virtualbox.box
```

Comment:
Those are compose links. Watch https://issues.redhat.com/browse/CS-2634 for when these will be officially available on cloud.centos.org